po/es.po

```diff
@@ -81,158 +80,150 @@ msgstr "_Cancelar"
 #: src/accountdialog.ui:22 src/remotedesktopdialog.ui:28
 #: src/screencastdialog.ui:28 src/screenshotdialog.ui:38
 msgid "_Share"
-msgstr "_Compartir"
+msgstr "Co_mpartir"
```

Comment:
Why changing the accelerator?

Reply:
I imagined that it would conflict with `_Cancelar`.